GameData/ROEngines/PartConfigs/RD200_RN.cfg

```diff
@@ -0,0 +1,81 @@
+PART
+{
+	module = Part
+	name = ROE-RD200
+	author = RaiderNick, Pap, Capkirk
+	
+	RSSROConfig = true
+	ROESetEngineDefaults = LIQUID_ENGINE
+	
+	//  ============================================================================
+	//	Update Below
+	//  ============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/RN/kosmos2_rd214
+		scale = 1.0, 1.0, 1.0
+	}
+	
+	scale = 1.0
+	rescaleFactor = 0.353
+	node_stack_top = 0.0, 1.0, 0, 0.0, 1.0, 0.0, 2
+	node_stack_bottom = 0.0, -0.830671, 0.0, 0, -1, 0, 2
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0
+	
+	title = RD-200
+	manufacturer = abc
+	description = abc
+	
+	tags = rd200, glushko
+	
+	engineType = RD200
+	clusterMultiplier = 1
+	
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+	}
+	
+	MODULE
+	{
+		name = FXModuleAnimateThrottle
+		animationName = FIRE
+		responseSpeed = 0.001
+		dependOnEngineState = True
+		dependOnThrottle = True
+	}
+	
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = ModuleGimbal
```

Comment:
Is this intentional to have the 2 gimbal declarations?

Reply:
Yes. One is for the Gimbal animation, and the other is for the Gimbal vector

Reply:
Are they decoupled, or is `thrustTransform` parented to `gimbal` so that steering `gimbal` implicitly governs thrust?  Should they have different range settings, or the same range settings, and does this need to set `enableRoll/enableYaw/enablePitch` = false if thrust _is_ parented to gimbal?